server/src/services/validation/HardhatWorker.ts

```diff
@@ -93,6 +97,9 @@ export class HardhatWorker implements WorkerProcess {
             this.logger.trace(
               `initialisation complete for ${this.project.basePath}`
             );
+            this.connection.sendNotification("custom/worker-initialized", {
+              project: this.project,
```

Comment:
Let's change the passed object to `{projectBasePath: string}`, in case we add something non-serializable to project without realizing it and break this.